test/functional/legacy/quickfix_spec.lua

```diff
@@ -2,16 +2,294 @@
 
 local helpers = require('test.functional.helpers')
 local source, clear = helpers.source, helpers.clear
+local execute, nvim, call = helpers.execute, helpers.meths, helpers.call
+
+local function expected_empty()
```

Comment:
It's supposed to be, but I removed the calls when I was trying to figure out a hang.